USF-1911: Adapted PlaceOrder container into the Checkout drop-in (#165)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/checkout/containers/place-order.mdx

  compact
  options={[
    ['Option', 'Type', 'Req?', 'Description'],
    ['disabled', 'boolean', 'No', 'Disables the Place Order button.'],

>     ['handleValidation', 'function', 'No', 'A function that performs validation checks and returns a boolean indicating whether the validation passed; in case of return false, the order will not be placed.'],

Reviewer: ```suggestion
    ['handleValidation', 'function', 'No', 'A function that performs validation checks and returns a boolean indicating whether the validation passed; in case it returns false, the order will not be placed.'],
```

    ['handlePlaceOrder', 'function', 'Yes', 'An asynchronous function that handles the order placement process. It receives a context object containing the selected payment method code and the cart ID.'],
  ]}
/>

## Example

> The following example renders the `PlaceOrder` container on a checkout page using the Adobe Payment Services as payment method. It includes functionality to validate login, shipping, and billing forms before placing an order. If the validation passes, it attempts to place the order and handles any errors that occur during the process.

Reviewer 2: ```suggestion
The following example renders the `PlaceOrder` container on a checkout page using the Payment Services drop-in component as a payment method. It includes functionality to validate login, shipping, and billing forms before placing an order. If the validation passes, it attempts to place the order and handles any errors.
```

    const shippingForm = forms[SHIPPING_FORM_NAME];

    if (
      success
      && shippingFormRef.current
      && shippingForm
      && shippingForm.checkVisibility()
    ) {
      success = shippingFormRef.current.handleValidationSubmit(false);
    }

    const billingForm = forms[BILLING_FORM_NAME];

    if (
      success
      && billingFormRef.current
      && billingForm
      && billingForm.checkVisibility()
    ) {
      success = billingFormRef.current.handleValidationSubmit(false);
    }

    return success;
  },
  handlePlaceOrder: async ({ code, cartId }) => {
    await displayOverlaySpinner();
    try {
      // Submit Payment Services credit card form
      if (code === CREDIT_CARD_CODE) {
        await paymentServicesCreditCard.submit();
      }
      // Place order
      await orderApi.placeOrder(cartId);
    } catch (error) {
      console.error(error);
      throw error;
    } finally {

>       await removeOverlaySpinner();

Reviewer: I don't think this `await` is needed.

Author: Removed, although this piece of code was extracted from the commerce-checkout-payment-services.js file.
